Combined patch to update the tutorial in the new design with many of the issues/pull requests for the old design

yatil opened the pull request:
Steps to add the tutorials to the website in the new layout and with the fixes outlined below fixed:

1. EOWG review and confirm changes in the netlify preview
2. Make changes to this PR if necessary. (It’s not. Changes are either minor editorial, by accessibility experts, or come from AG WG or its sub-groups.)
3. Merge this PR.
4. Merge PR https://github.com/w3c/wai-website/pull/283
5. Merge PR https://github.com/w3c/wai-website-theme/pull/39
5. Merge PR https://github.com/w3c/wai-website-data/pull/81
6. Success

## Included fixes:

- updating metadata https://github.com/w3c/wai-tutorials/commit/4b748074feb0d3ebd5251f3d5e5f03c88cf0e09b
- Fixes w3c/wai-tutorials#628
- Fixes w3c/wai-tutorials#622
- Fixes w3c/wai-tutorials#632 (Typo in image groups)
- Fixes w3c/wai-tutorials#633
- Fixes w3c/wai-tutorials#620
- Fixes w3c/wai-tutorials#590 and fixes w3c/wai-tutorials#606
- Fixes w3c/wai-tutorials#626
- Fixes w3c/wai-tutorials#618
- Fixes w3c/wai-tutorials#629
- Fixes w3c/wai-tutorials#631

---

[Diff for the flyout menus page.](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FWAI%2Ftutorials%2Fmenus%2Fflyout%2F&doc2=https%3A%2F%2Fdeploy-preview-636--wai-tutorials.netlify.app%2Ftutorials%2Fmenus%2Fflyout%2F)

yatil committed:
Update submodule

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Use includes instead of indexOf

Addresses w3c/wai-tutorials#628

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Remove unnecessary submit button (likely copy/paste error)

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Fix id->for typo

Fixes w3c/wai-tutorials#622. Thanks @equipaje!

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Fix typo in image groups

See: w3c/wai-tutorials#632
Thanks @jake-crawford-nic

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Fix flyout menu (=navigation) page

Thanks a million to @scottaohara who did all the work in w3c/wai-tutorials#633

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Fix approach reference

See w3c/wai-tutorials#620

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Clarify header and footer when not scoped to the root element

Fixes w3c/wai-tutorials#590 and w3c/wai-tutorials#606. Thanks to @JAWS-test2 for suggesting clarifications.

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Simplify ::placeholder code

Thanks @hidde for the suggestion in w3c/wai-tutorials#626

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Remove reference to margin

Fixes w3c/wai-tutorials#618. Thanks to anonymous emailer.

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Replace “full-text equivalent” with “complete text equivalent”

Fixes w3c/wai-tutorials#629

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Better wording and including subject matter experts

Fixes w3c/wai-tutorials#631.

Thank you @MakotoUeki for the suggestion.

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil commented:
This commit broke mergability: https://github.com/w3c/wai-tutorials/commit/b8a54e37f196d3b92595f2d3c97eddbc93fb1ba9 – I will merge the changes in.

yatil committed:
Merge branch 'master-2.0' of https://github.com/w3c/wai-tutorials into w3c-master-2.0

yatil committed:
Update data

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil committed:
Merge branch 'w3c-master-2.0' into patch-update-wai

shawna-slh commented:
Hey @yatil

Would you take a look at the footers?
* [ ] The Updated date isn’t showing up for me.
* [ ] space missing between “Eggertand”
* [ ] is “Testy Tester” correct? It’s on the first page.
* [ ] new format. If you’re up for updating the format, that’d be nice. But not at all essential. new format:
```
<p><strong>Date:</strong> Updated @@ Month 2022. First published Month 20@@. <a href="https://deploy-preview-636--wai-tutorials2.netlify.app/tutorials/changelog/">Changelog</a>.</p>
<p><strong>Editors:</strong> @@name, @@name. Contributors: @@name, @@name, and <a href="https://www.w3.org/groups/wg/eowg/participants">participants of the EOWG</a>.</p>
<p>Developed by the Accessibility Education and Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed as part of the <a href="https://www.w3.org/WAI/ACT/">WAI-ACT project</a>, co-funded by the European Commission.</p>
```
<hr>

* [ ] Also looks like changelog needs to be updated.

Thanks a million for stepping in to get this out!!!

yatil committed:
Fix minor issues

- Remove testy tester
- fix metafooter outputting last updated date

Signed-off-by: Eric Eggert <mail@yatil.net>

yatil commented:
The footer for the tutorials are using a different format than other documents on the WAI website to ease with transition from the old website. Feel free to change them to the version of other documents.

> - The Updated date isn’t showing up for me.
> - space missing between “Eggertand”

I fixed the “metafooter” accordingly and also added this Pull Request for the Theme to display the footer when it is published:

- https://github.com/w3c/wai-website-theme/pull/39

> - is “Testy Tester” correct? It’s on the first page.

No, this was a dummy entry to test some CSS. Removed in:

- https://github.com/w3c/wai-tutorials/pull/636/commits/6f95d3f9342f40f57c91835995bfa4fe5bfe779c

> - new format. If you’re up for updating the format, that’d be nice. But not at all essential. new format:

The format can be edited in the metafooter part of the include here. As it is generated, and using the old format for easier porting, I have not looked into different formats. The format here uses semantic lists, nesting them into p elements is not possible as by HTML spec, so logic/formatting needs to change. Happy for y‘all to make the change.

- File for preview in this repository: https://github.com/yatil/wai-tutorials/blob/patch-update-wai/_includes/footer.html
- File in the wai-website-theme repository that would be used by the full site: https://github.com/yatil/wai-website-theme/blob/wai-patch/_includes/footer.html

I have included the https://github.com/w3c/wai-website-theme/pull/39 Pull Request in the list of steps in my initial comment.

shawna-slh commented:
Thanks, @yatil
